pkg/scheduler/util/utils.go

```diff
@@ -67,12 +67,12 @@ func GetEarliestPodStartTime(victims *extenderv1.Victims) *metav1.Time {
 	maxPriority := corev1helpers.PodPriority(victims.Pods[0])
 
 	for _, pod := range victims.Pods {
-		if corev1helpers.PodPriority(pod) == maxPriority {
-			if GetPodStartTime(pod).Before(earliestPodStartTime) {
-				earliestPodStartTime = GetPodStartTime(pod)
+		if tmpPriority := corev1helpers.PodPriority(pod); tmpPriority == maxPriority {
+			if tmpStartTime := GetPodStartTime(pod); tmpStartTime.Before(earliestPodStartTime) {
+				earliestPodStartTime = tmpStartTime
 			}
-		} else if corev1helpers.PodPriority(pod) > maxPriority {
-			maxPriority = corev1helpers.PodPriority(pod)
+		} else if tmpPriority > maxPriority {
+			maxPriority = tmpPriority
 			earliestPodStartTime = GetPodStartTime(pod)
 		}
```

Comment:
nits: please rename tmpPriority -> podPriority and tmpStartTime -> podStartTime. That's straightforward.

Reply:
ok,thanks